### ISSUE TITLE
Sandstorm should have a dark mode

### ISSUE THREAD

**Issue by ocdtrekkie** [sandstorm-io/sandstorm#3229]
This was requested on the email list. I am a big dark mode aficionado, so I'd love this.

Currently, browsers are starting to standardize around a way to inform websites of what they prefer, https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme which Firefox, Chrome, and Safari all support.

I am guessing we don't do anything to prevent this setting from carrying through to apps in the sandbox as well, but we should ideally support apps' detection and implementation of dark mode as well.

**Comment by ocdtrekkie** [comment]
This would probably be a really good newcomer project since, if based solely around the prefers-color-scheme behavior, it'd be solely an exercise in CSS to implement for the Sandstorm shell. Though, as with language, we should really have a user option to set it specifically for Sandstorm (#3044).

**Comment by codecowboy** [comment]
Is there a user story for this? And do you use some sort of kanban board? Would it be a Sandstorm setting for the shell and admin panel only? Or would it check if the individual grains also support it within each app?

**Comment by xet7** [comment]
@codecowboy 

Wekan kanban board already has dark mode, see 2nd screenshot at:
https://wekan.github.io/

So this is most likely about Sandstorm web UI only, shell and admin panel.

For other apps, I think it depends on individual grains which of them have dark mode.

**Comment by ocdtrekkie** [comment]
Yeah, we can’t define what apps do, but if we accept the browser’s preference and apps do as well, it’d be nice. Most major web apps either have introduced dark mode support or are working on it, so I suspect many self hosted apps will support it in the future as well.

I wouldn’t consider this a high priority feature by any means, but it would probably be an easy one for someone to get involved with.

**Comment by zenhack** [comment]
@codecowboy re: user stories, I don't think there's anything more formal than this issue right now. If you're asking about a kanban board we use to organize issues like this, not really (I think we tried having a "what people are working on," board, but I've lost track of where it was and I don't know if other folks have been updating or referencing it).

**Comment by codecowboy** [comment]
May I boldly suggest then that we start a new one?

**Comment by ocdtrekkie** [comment]
I can send you an invite link to it, it’s on Alpha. But not every issue/feature request people make gets an entry there.

**Comment by zenhack** [comment]
Getting slightly off-topic, but @ocdtrekkie has also expressed he'd like to start organizing things into higher level milestones and having more explicit goals. Maybe something to talk about on the next office hours call (which is tonight).
